.github/workflows/nm-build.yml

```diff
@@ -125,27 +125,6 @@ jobs:
                 python: ${{ inputs.python }}
                 venv: ${{ env.VENV_BASE }}
 
-            # GCP
-            - name: 'Authenticate to Google Cloud'
```

Comment:
please don't remove this. it is hear for us to have "longish" term backups.

Reply:
I added the code back but it looks duplicated some part of the code in workflows/nm-upload-assets-to-gcp.yml. Should we integrate the two together so it'll be better organized? I understand we're uploading the whl/tar.gz to a different gcp location than the nm pypi upload so maybe we can pass the gcp location as an argument to an action and both nm-build.yml and nm-upload-assets-to-gcp.yml can call?

Reply:
Resolving to keep the code as is.